apps/price_pusher/src/solana/solana.ts

```diff
@@ -28,6 +28,25 @@ export class SolanaPriceListener extends ChainPriceListener {
     super(config.pollingFrequency, priceItems);
   }
 
+  // Checking the health of the Solana connection by checking the last block time
+  // and ensuring it is not older than 30 seconds.
+  private async checkHealth() {
```

Comment:
should we have a health check for Jito?

Reply:
i throw an error on the failures there. Didn't do on Solana because it might fail due to landing issues.